Source/Fuse.Reactive.JavaScript/Docs/JavaScriptRemarks.md

```diff
@@ -43,6 +43,68 @@ Failing to export from modules will make it impossible to reach defined data ins
 
 This is good for hiding implementation details from other calling JavaScript modules and UX code.
 
+## Reactive modules
+
+A module related with a `<JavaScript>` tag supports the Reactive Modules API. This is a set of methods on `module` that allows the module to change the exported values and notifying the data context about the change. 
+
+Fuse's Reactive modules feature allows reactive programming between JavaScript modules and native code in a non-intrusive way. You are free to export arbitrary JavaScript objects from your module. Later, you can make observable changes to any part of this state without using observables (like Rx) or introducing a strict totalitarian state container (like Redux).
+
+Each of these APIs accept a `path` to the *node* in the `exports` tree. The path is a sequence of either string keys (to look up into objects) or number indices (to look up into arrays).
+
+### `module.set(... path, value)`
+
+Sets the value at the `path` to `value`. The `value` can be an arbitrary JavaScript object. 
```

Comment:
Only if you replace the entire root object, `module.set({ .. new object goes here ..})`